exercises/concept/card-tricks/.docs/introduction.md

```diff
@@ -28,27 +28,88 @@ If you don't specify an ending index, it defaults to the length of the slice.
 newSlice := withData[2:4] // newSlice == []int{2,3}
 ```
 
-## Multiple Return Values
+Go provides a built-in `append` function to append new elements to a slice: 
```

Comment:
I have replaced this with the phrase below from the official [Go Tour](https://go.dev/tour/moretypes/15). It is also quite similar to the formulation in [Go slices](https://go.dev/blog/slices-intro).  
```text
It is common to append new elements to a slice, and so Go provides a built-in `append` function.
```

Reply:
That's ok. In general though keep in mind that we try to be even more accessible than existing resources. For example in this case imagine a non-native speaker that might struggle with the word "append" for whatever reason (maybe the translations in their native language are ambiguous). For them it would be valuable to read append explained with "add to the end" instead of having "append" explained with "append".

Reply:
Ok, I will use the "add" explanation. I have added a [yourbasic](https://yourbasic.org/golang/append-explained/) link.  I like the explanation of `append` on the web site. Is it an appropriate link with regards to our "linking policies"?

Reply:
I don't see a reason why not as long as the content is correct.